### ISSUE TITLE
Visual Studio Python Environment missing in IronPython 2.7.8 but works in 2.7.7

### ISSUE THREAD

**Issue by GLantaff** [Microsoft/PTVS#4092]
If I install IP 2.7.7, I can create IronPython projects and VS will find the "IronPython 2.7 (32-bit)" Python Environment.
If I then uninstall 2.7.7 and install 2.7.8, The IronPython Environments are gone.

Any thoughts?

**Comment by huguesv** [comment]
They changed the location from `Program Files (x86)` to `Program Files`, and the registry key from `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\IronPython\2.7\InstallPath` to `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\IronPython\2.7\InstallPath`.

Workaround: Using `regedit.exe`, add a copy of the `IronPython\2.7\InstallPath` under `WOW6432Node`. I checked that it was detected and I was able to run a couple of IronPython projects after that.
